docs/english/reference/web-api/classes/WebClient.md

```diff
@@ -4625,7 +4625,7 @@ Stream markdown text into a conversation.
 
 #### Description
 
-The "chatStream" method starts a new chat stream in a coversation that can be appended to. After appending an entire message, the stream can be stopped with concluding arguments such as "blocks" for gathering feedback.
+The "chatStream" method starts a new chat stream in a conversation that can be appended to. After appending an entire message, the stream can be stopped with concluding arguments such as "blocks" for gathering feedback.
```

Comment:
👁️‍🗨️ note: I fear this change might also be needed in the Python implementation... Will check!